cocoa/src/toga_cocoa/app.py

```diff
@@ -338,6 +340,15 @@ def main_loop(self):
 
         self.loop.run_forever(lifecycle=CocoaLifecycle(self.native))
 
+    def get_screens(self):
+        screens = []
+        native_screens = NSScreen.screens
+        for screen in native_screens:
+            _impl = ImplScreen(native=screen)
+            screen_class = InterfaceScreen(_impl=_impl)
+            screens.append(screen_class)
```

Comment:
This does 2 hash lookups to resolve the impl and interface instance - but once you've got the impl, you *should*  be able to ask for `impl.interface`. However, that linkage hasn't been made in the construction chain.